Formula/apache-archiva.rb

```diff
@@ -4,17 +4,28 @@ class ApacheArchiva < Formula
   url "https://www.apache.org/dyn/closer.lua?path=archiva/2.2.5/binaries/apache-archiva-2.2.5-bin.tar.gz"
   mirror "https://archive.apache.org/dist/archiva/2.2.5/binaries/apache-archiva-2.2.5-bin.tar.gz"
   sha256 "01119af2d9950eacbcce0b7f8db5067b166ad26c1e1701bef829105441bb6e29"
-  license "Apache-2.0"
+  license all_of: ["Apache-2.0", "GPL-2.0-only"]
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "7be56a181bec0957d8f4f2936e99a98f737b3f7e146a3f7384b13d9e824919ad"
   end
 
+  depends_on "ant" => :build
+  depends_on "java-service-wrapper" => :build
```

Comment:
Can we use symlinks instead of copying stuff over, and make this a runtime dependency?

Reply:
Done